src/read_write_tensorstore.py

```diff
@@ -0,0 +1,79 @@
+from typing import Literal
+import tensorstore as ts
+import numpy as np
+import pathlib
+import utils
+
+
+def read_zarr_array(store_path: pathlib.Path) -> np.array:
+    """Read the v2 zarr spec with tensorstore"""
+    zarr_read = ts.open(
+        {
+            "driver": "zarr",
+            "kvstore": {
+                "driver": "file",
+                "path": str(store_path),
```

Comment:
```suggestion
                "path": str(store_path.resolve()),
```

Maybe, to resolve relative paths first?